x-pack/test/alerting_api_integration/security_and_spaces/tests/alerting/alerts.ts

```diff
@@ -263,20 +282,26 @@ export default function alertTests({ getService }: FtrProviderContext) {
             case 'space_1_all at space1':
               expect(response.statusCode).to.eql(200);
               objectRemover.add(space.id, response.body.id, 'alert');
-              alertTestRecord = (
-                await esTestIndexTool.waitForDocs('alert:test.authorization', reference)
-              )[0];
-              expect(alertTestRecord._source.state).to.eql({
+
+              // Wait for test.authroization to index a document before disabling the alert and waiting for tasks to finish
```

Comment:
`authorization` is misspelled here, and once again below this

Reply:
Good catch! Fixed in https://github.com/elastic/kibana/pull/50246/commits/d0c43c4fb93fd2212be2916b9bc86fa4df37237c.